share/governance.md

```diff
@@ -105,10 +113,12 @@ election. There are 5 members of the PMC. Terms are 1 year.
 
 The PMC doesn't have specific roles or a chairperson.
 
-At the moment, IRC is our main communication channel. PMC members should be
-present in our `#voxpupuli-pmc` channel on [libera.chat](https://libera.chat).
+IRC is the main communication channel for PMC business. PMC members should be
+present in our `#voxpupuli-pmc` channel on [libera.chat](https://libera.chat)
+or the connected Slack channel on the Vox Pupuli slack workspace.
+
 Also we have some gpg-encrypted credentials and a few repositories require
-gpg-signed commits, so a PMC member also needs a gpg key.
+gpg-signed commits, so a PMC member also needs a GPG key.
```

Comment:
That's not correct anymore, could you please update it? Some repositories require signed commits, they can be signed with an ssh key or a GPG key.

Reply:
This seems unrelated to governance and just a practical part of onboarding for the type of key to access a repo. I just changed it for style.